home/ivp-planning/settings/library-settings.md

```diff
@@ -0,0 +1,12 @@
+# Library
+
+The library settings enable you to locate and modify your local library path. Additionally, they also grant you the choice to selectively load or exclude assets into the program.
```

Comment:
Loading/excluding should be explained in more detail:

* Users won't know what you mean with assets
* Need to mention that objects come in bundles and you can modify the entire bundle, but not individual objects
* Need to explain the individual options
* Need to explain the benefit of not loading some bundles (updates, startup speed, disk space (when deleted))